manifests/j/JabRef/JabRef/5.3/JabRef.JabRef.yaml

```
@@ -0,0 +1,22 @@
+PackageIdentifier: JabRef.JabRef
+Publisher: JabRef
+PackageName: JabRef
+PackageVersion: 5.3
```

```suggestion
PackageVersion: 5.3.60000
```
Windows Package Manager now requires this key to match Control Panel for the best application upgrade experience.